x-pack/plugins/index_lifecycle_management/public/sections/edit_policy/components/min_age_input.js

```diff
@@ -72,8 +57,20 @@ const MinAgeInputUi = props => {
           />
         </ErrableFormRow>
       </EuiFlexItem>
-      <EuiFlexItem style={{ maxWidth: 188 }}>
-        <EuiFormRow hasEmptyLabelSpace>
+      <EuiFlexItem style={{ maxWidth: 220 }}>
+        <EuiFormRow
+          label={
+            <LearnMoreLink
+              docPath="_timing.html"
+              text={
+                <FormattedMessage
+                  id="xpack.indexLifecycleMgmt.editPolicy.learnAboutTimingText"
+                  defaultMessage="Learn about timing"
```

Comment:
Can we make this `helpText` for the first input? Typically clicking on a label gives an input focus, so this it's a bit surprising to make it a link instead.

![image](https://user-images.githubusercontent.com/1238659/49971481-8825a580-fee3-11e8-93bb-67605fb8b97d.png)

Reply:
makes sense, done